modules/healthconnectonfhir/src/test/java/edu/stanford/healthconnectonfhir/RecordToObservationMapperTests.kt

```diff
@@ -0,0 +1,195 @@
+package edu.stanford.healthconnectonfhir
+
+import androidx.health.connect.client.records.ActiveCaloriesBurnedRecord
+import androidx.health.connect.client.records.BloodPressureRecord
+import androidx.health.connect.client.records.BodyTemperatureRecord
+import androidx.health.connect.client.records.HeartRateRecord
+import androidx.health.connect.client.records.HeightRecord
+import androidx.health.connect.client.records.StepsRecord
+import androidx.health.connect.client.records.WeightRecord
+import androidx.health.connect.client.units.Energy
+import androidx.health.connect.client.units.Length
+import androidx.health.connect.client.units.Mass
+import androidx.health.connect.client.units.Pressure
+import androidx.health.connect.client.units.Temperature
+import ca.uhn.fhir.context.FhirContext
+import com.google.common.truth.Truth.assertThat
+import org.hl7.fhir.r4.model.Observation
+import org.hl7.fhir.r4.model.Period
+import org.hl7.fhir.r4.model.Quantity
+import org.junit.Before
+import org.junit.Test
+import java.time.Instant
+import java.time.ZoneOffset
+import java.util.Date
+
+class RecordToObservationMapperTests {
+
+    private val fhirContext: FhirContext = FhirContext.forR4()
```

Comment:
We can remove this as it is not being used :)

Minor: At line 30 we can define the mapper directly as `private val mapper = RecordToObservationMapperImpl()`. First the mapper is stateless as there is no need to recreate it before every test. Also, JUnit recreates test class instances and runs each test method on a new instance :) (see [reference](https://www.baeldung.com/junit-testinstance-annotation#introduction) )